src/components/FilterPanel/FilterPanel.scss

```diff
@@ -0,0 +1,27 @@
+.filter-section {
+    ul {
```

Comment:
Always try to write css using classes instead of elements since this element can be used throughout the page and will affect the css of latter one.